File: naive.py
import time

def main():
    base_string = input()
    number_of_queries , query_length = list(map(int, input().split()))
    base_string_length = len(base_string)

    queries = [input() for i in range(number_of_queries)]


    for i in range (number_of_queries):
        query = queries[i]
        flag = False
        for j in range(base_string_length-query_length-1):
            if(base_string[j:j+query_length] == query):
                print(f"The sequence: {query} found at the position: {base_string.find(query) + 1}")
                flag = True

        if(not flag):
            print(f"The sequence: {query} not found !")


if __name__ == "__main__":
    start_time = time.time()
    main()
    print( "The program execution time in ms: " + str(int((time.time() - start_time) * 1000)))

File: Naive.java
import java.util.ArrayList;
import java.util.Scanner;

public class Main2 {
    public static int numberOfQueries;
    public static int queryLength;
    public static int baseStringLength;
    public static String baseString;
    public static String query;
    static ArrayList<String> queries = new ArrayList<String>();

    public static void main(String[] args) {

        long startTime = System.nanoTime();

        Scanner input = new Scanner(System.in);
        baseString = input.next();
        numberOfQueries = input.nextInt();
        queryLength = input.nextInt();
        baseStringLength = baseString.length();
        for (int i = 0; i < numberOfQueries; i++) {
            queries.add(input.next());
        }

        for (int i = 0; i < numberOfQueries; i++) {
            query = queries.get(i);
            boolean flag = false;
            for(int j=0; j<(baseStringLength-queryLength-1); j++){
                if(baseString.substring(j, j + queryLength).equals(query)){
                    System.out.println("The sequence: '" + query + "' found at the position: " + (baseString.indexOf(query)+1) );
                    flag = true;
                    break;
                }
            }

            if (!flag){
                System.out.println("The sequence: '" + query + "' not found!");
            }
        }

        long endTime = System.nanoTime();
        long duration = (endTime - startTime) / 1000000;
        System.out.println("The program execution time in ms: "+duration);

    }
}
